// @wix/stores_customizations-v-3/build/cjs/src/stores-catalog-v3-customization-customizations-v-3.types.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.WebhookIdentityType = exports.SortOrder = exports.RequestedFields = exports.ChoiceType = exports.CustomizationRenderType = exports.CustomizationType = void 0;
var CustomizationType;
(function (CustomizationType) {
    CustomizationType["UNKNOWN_CUSTOMIZATION_TYPE"] = "UNKNOWN_CUSTOMIZATION_TYPE";
    CustomizationType["PRODUCT_OPTION"] = "PRODUCT_OPTION";
    CustomizationType["MODIFIER"] = "MODIFIER";
})(CustomizationType || (exports.CustomizationType = CustomizationType = {}));
var CustomizationRenderType;
(function (CustomizationRenderType) {
    CustomizationRenderType["UNKNOWN_CUSTOMIZATION_RENDER_TYPE"] = "UNKNOWN_CUSTOMIZATION_RENDER_TYPE";
    CustomizationRenderType["FREE_TEXT"] = "FREE_TEXT";
    CustomizationRenderType["TEXT_CHOICES"] = "TEXT_CHOICES";
    CustomizationRenderType["SWATCH_CHOICES"] = "SWATCH_CHOICES";
})(CustomizationRenderType || (exports.CustomizationRenderType = CustomizationRenderType = {}));
var ChoiceType;
(function (ChoiceType) {
    ChoiceType["UNKNOWN_CHOICE_TYPE"] = "UNKNOWN_CHOICE_TYPE";
    ChoiceType["CHOICE_TEXT"] = "CHOICE_TEXT";
    ChoiceType["ONE_COLOR"] = "ONE_COLOR";
    ChoiceType["MULTIPLE_COLORS"] = "MULTIPLE_COLORS";
    ChoiceType["IMAGE"] = "IMAGE";
})(ChoiceType || (exports.ChoiceType = ChoiceType = {}));
var RequestedFields;
(function (RequestedFields) {
    RequestedFields["UNKNOWN_REQUESTED_FIELD"] = "UNKNOWN_REQUESTED_FIELD";
    RequestedFields["ASSIGNED_PRODUCTS_COUNT"] = "ASSIGNED_PRODUCTS_COUNT";
})(RequestedFields || (exports.RequestedFields = RequestedFields = {}));
var SortOrder;
(function (SortOrder) {
    SortOrder["ASC"] = "ASC";
    SortOrder["DESC"] = "DESC";
})(SortOrder || (exports.SortOrder = SortOrder = {}));
var WebhookIdentityType;
(function (WebhookIdentityType) {
    WebhookIdentityType["UNKNOWN"] = "UNKNOWN";
    WebhookIdentityType["ANONYMOUS_VISITOR"] = "ANONYMOUS_VISITOR";
    WebhookIdentityType["MEMBER"] = "MEMBER";
    WebhookIdentityType["WIX_USER"] = "WIX_USER";
    WebhookIdentityType["APP"] = "APP";
})(WebhookIdentityType || (exports.WebhookIdentityType = WebhookIdentityType = {}));
