// @wix/stores_customizations-v-3/build/es/src/stores-catalog-v3-customization-customizations-v-3.types.js
export var CustomizationType;
(function (CustomizationType) {
    CustomizationType["UNKNOWN_CUSTOMIZATION_TYPE"] = "UNKNOWN_CUSTOMIZATION_TYPE";
    CustomizationType["PRODUCT_OPTION"] = "PRODUCT_OPTION";
    CustomizationType["MODIFIER"] = "MODIFIER";
})(CustomizationType || (CustomizationType = {}));
export var CustomizationRenderType;
(function (CustomizationRenderType) {
    CustomizationRenderType["UNKNOWN_CUSTOMIZATION_RENDER_TYPE"] = "UNKNOWN_CUSTOMIZATION_RENDER_TYPE";
    CustomizationRenderType["FREE_TEXT"] = "FREE_TEXT";
    CustomizationRenderType["TEXT_CHOICES"] = "TEXT_CHOICES";
    CustomizationRenderType["SWATCH_CHOICES"] = "SWATCH_CHOICES";
})(CustomizationRenderType || (CustomizationRenderType = {}));
export var ChoiceType;
(function (ChoiceType) {
    ChoiceType["UNKNOWN_CHOICE_TYPE"] = "UNKNOWN_CHOICE_TYPE";
    ChoiceType["CHOICE_TEXT"] = "CHOICE_TEXT";
    ChoiceType["ONE_COLOR"] = "ONE_COLOR";
    ChoiceType["MULTIPLE_COLORS"] = "MULTIPLE_COLORS";
    ChoiceType["IMAGE"] = "IMAGE";
})(ChoiceType || (ChoiceType = {}));
export var RequestedFields;
(function (RequestedFields) {
    RequestedFields["UNKNOWN_REQUESTED_FIELD"] = "UNKNOWN_REQUESTED_FIELD";
    RequestedFields["ASSIGNED_PRODUCTS_COUNT"] = "ASSIGNED_PRODUCTS_COUNT";
})(RequestedFields || (RequestedFields = {}));
export var SortOrder;
(function (SortOrder) {
    SortOrder["ASC"] = "ASC";
    SortOrder["DESC"] = "DESC";
})(SortOrder || (SortOrder = {}));
export var WebhookIdentityType;
(function (WebhookIdentityType) {
    WebhookIdentityType["UNKNOWN"] = "UNKNOWN";
    WebhookIdentityType["ANONYMOUS_VISITOR"] = "ANONYMOUS_VISITOR";
    WebhookIdentityType["MEMBER"] = "MEMBER";
    WebhookIdentityType["WIX_USER"] = "WIX_USER";
    WebhookIdentityType["APP"] = "APP";
})(WebhookIdentityType || (WebhookIdentityType = {}));
