// @wix/stores_inventory/build/es/src/stores-v2-inventory-inventory.context.js
import { getInventoryVariants as publicGetInventoryVariants, queryInventory as publicQueryInventory, updateInventoryVariants as publicUpdateInventoryVariants, decrementInventory as publicDecrementInventory, incrementInventory as publicIncrementInventory, } from './stores-v2-inventory-inventory.public';
import { createRESTModule } from '@wix/sdk-runtime/rest-modules';
import { createEventModule } from '@wix/sdk-runtime/event-definition-modules';
import { onInventoryItemChanged as publicOnInventoryItemChanged } from './stores-v2-inventory-inventory.public';
import { onInventoryVariantsChanged as publicOnInventoryVariantsChanged } from './stores-v2-inventory-inventory.public';
export const getInventoryVariants = /*#__PURE__*/ createRESTModule(publicGetInventoryVariants);
export const queryInventory = /*#__PURE__*/ createRESTModule(publicQueryInventory);
export const updateInventoryVariants = /*#__PURE__*/ createRESTModule(publicUpdateInventoryVariants);
export const decrementInventory = /*#__PURE__*/ createRESTModule(publicDecrementInventory);
export const incrementInventory = /*#__PURE__*/ createRESTModule(publicIncrementInventory);
export { publicOnInventoryItemChanged };
/**
 * Triggered when an inventory item is changed.
 */
export const onInventoryItemChanged = createEventModule(publicOnInventoryItemChanged);
export { publicOnInventoryVariantsChanged };
/**
 * Triggered when inventory variants are changed.
 */
export const onInventoryVariantsChanged = createEventModule(publicOnInventoryVariantsChanged);
export { ReasonType, WebhookIdentityType, } from './stores-v2-inventory-inventory.universal';
